packages/vaadin-template-renderer/src/vaadin-template-renderer-templatizer.js

```diff
@@ -56,6 +64,18 @@ export class Templatizer extends PolymerElement {
     return instance;
   }
 
+  __disposeOfTemplateInstance(instance) {
+    this.__templateInstances.delete(instance);
+  }
+
+  __hasTemplateInstance(instance) {
+    return this.__templateInstances.has(instance);
+  }
+
+  __isTemplateInstanceConnected(instance) {
+    return instance.children[0].isConnected;
```

Comment:
Better check for parent than isConnected because isConnected would be false if grid itself was temporarily disconnected from the DOM

Reply:
Oh, you are right, I didn't take this case into account. Thanks for the good catch! Fixed.